examples/config/test-macro.hpp

```diff
@@ -1,28 +1,37 @@
-/////////////////////////////////////////////////////////////////////////////////
-//  This is an auto-generated file that defines a set of configuration options.
-//
-//  To create a new config from scratch, the format is:
-//    EMP_BUILD_CONFIG( CLASS_NAME, OPTIONS... )
-//
-//  To extend an existing config, simply use:
-//    EMP_EXTEND_CONFIG( NEW_NAME, BASE_CLASS, OPTIONS... )
-//
-//  The available OPTIONS are:
-//
-//  GROUP(group name, group description string)
-//   Start a new group of configuration options.  Group structure is preserved
-//   when user-accessible configuration options are generated.
-//
-//  VALUE(variable name, type, default value, description string)
-//   Create a new setting in the emp::Config object that can be easily accessed.
-//
-//  CONST(variable name, type, fixed value, description string)
-//   Create a new configuration constant that cannot be changed.  In practice,
-//   allows broader optimizations in the code.
-//
-//  ALIAS(alias name)
-//   Include an alias for the previous setting.  This command is useful to
-//   maintain backward compatibility if names change in newer software versions.
+/**
+ *  @note This file is part of Empirical, https://github.com/devosoft/Empirical
+ *  @copyright Copyright (C) Michigan State University, MIT Software license; see doc/LICENSE.md
+ *  @date 2021
+ *
+ *  @file test-macro.hpp
+ *  @brief This is an auto-generated file that defines a set of configuration options.
+ *  To create a new config from scratch, the format is:
+ *    EMP_BUILD_CONFIG( CLASS_NAME, OPTIONS... )
+ *
+ *  To extend an existing config, simply use:
+ *    EMP_EXTEND_CONFIG( NEW_NAME, BASE_CLASS, OPTIONS... )
+ *
+ *  The available OPTIONS are:
+ *
+ *  GROUP(group name, group description string)
+ *   Start a new group of configuration options.  Group structure is preserved
+ *   when user-accessible configuration options are generated.
+ *
+ *  VALUE(variable name, type, default value, description string)
+ *   Create a new setting in the emp::Config object that can be easily accessed.
+ *
+ *  CONST(variable name, type, fixed value, description string)
+ *   Create a new configuration constant that cannot be changed.  In practice,
+ *   allows broader optimizations in the code.
+ *
+ *  ALIAS(alias name)
+ *   Include an alias for the previous setting.  This command is useful to
+ *   maintain backward compatibility if names change in newer software versions.
+ */
+
+#pragma once
```

Comment:
It looks like these #pragma once declarations are being added now?  I suspect it's more likely an issue with how github is displaying the diffs...  in any case, identifying #pragma and recommending against it might be a good script in general...

Reply:
Good catch. Yeah I’ll swap those out and add a simple script that searches for pragma once and gives a warning.

Reply:
Just pushed a fix for this. Autogenerates headerguards for material outside of include/ now